### PR TITLE
Issue #83

### PR DESCRIPTION
**Pull request by jeestr4d**
- **Added logic for CSCvn20175 and CSCvt07565**
- **Added logic for CSCvn20175 and CSCvt07565**
- **Added logic for CSCvn20175 and CSCvt07565**

### CODE REVIEW

**Comment by jeestr4d** [comment]
Fix for Issue #83

**Review by monrog2 — CHANGES_REQUESTED** [review]
1. paste in the output of your local pytest run for this function
2. The associated validation doc likely need an update to mention the new bug.

File: aci-preupgrade-validation-script.py [hunk 1]
```diff
@@ -2328,12 +2328,16 @@ def eventmgr_db_defect_check(index, total_checks, cversion, **kwargs):
     recommended_action = 'Contact Cisco TAC to check the DB size via root'
     print_title(title, index, total_checks)
 
-    if cversion.older_than('3.2(5d)') or (cversion.major1 == '4' and cversion.older_than('4.1(1i)')):
+    if cversion.older_than('3.2(5d)') or (cversion.major1 == '4' and cversion.major2 <= '1' and cversion.older_than('4.1(1i)')):  # If current Version is older than 3.2(5d) or for 4.1 version is older than 4.1.1i
         result = FAIL_UF
         data.append(['CSCvn20175', recommended_action])
 
+    elif (cversion.major1 == '4' and cversion.major2 == '2' and cversion.older_than('4.2(4i)')) or (cversion.major1 == '5' and cversion.older_than('5.0(1k)')):  # If current 4.2 Version is older than 4.2(4i) or for 5.X version is older than 5.0(1k)
```
[the remark this answers is not in the document]
Comment (jeestr4d) [reply aci-preupgrade-validation-script.py]: 1. Local pytest function run
`tests/eventmgr_db_defect_check/test_eventmgr_db_defect_check.py::test_logic[3.2(5e)-PASS] PASSED                                         [ 20%]
tests/eventmgr_db_defect_check/test_eventmgr_db_defect_check.py::test_logic[3.2(10g)-PASS] PASSED                                        [ 21%]
tests/eventmgr_db_defect_check/test_eventmgr_db_defect_check.py::test_logic[4.1(2a)-PASS] PASSED                                         [ 21%]
tests/eventmgr_db_defect_check/test_eventmgr_db_defect_check.py::test_logic[4.2(4j)-PASS] PASSED                                         [ 22%]
tests/eventmgr_db_defect_check/test_eventmgr_db_defect_check.py::test_logic[5.0(1l)-PASS] PASSED                                         [ 22%]
tests/eventmgr_db_defect_check/test_eventmgr_db_defect_check.py::test_logic[3.2(4a)-FAIL - UPGRADE FAILURE!!] PASSED                     [ 23%]
tests/eventmgr_db_defect_check/test_eventmgr_db_defect_check.py::test_logic[4.0(1a)-FAIL - UPGRADE FAILURE!!] PASSED                     [ 23%]
tests/eventmgr_db_defect_check/test_eventmgr_db_defect_check.py::test_logic[4.2(3a)-FAIL - UPGRADE FAILURE!!] PASSED                     [ 23%]
tests/eventmgr_db_defect_check/test_eventmgr_db_defect_check.py::test_logic[5.0(1a)-FAIL - UPGRADE FAILURE!!] PASSED                     [ 24%]
tests`
2. Branch updated to include doc_url
3. Removed comments

**Comment by monrog2** [comment]
closing this one, will work on branch sourced from here.

**Pull request closed without merging**